tests/chainer_tests/datasets_tests/test_concatenated_dataset.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+import numpy as np
+
+from chainer.datasets import ConcatenatedDataset
+from chainer import testing
+
+
+class TestConcatenatedDataset(unittest.TestCase):
```

Comment:
Please add these tests:
* empty list of base dataset is given (`ConcatenatedDataset()`)
* only a single base dataset is given (`ConcatenatedDataset(dataset0)`)
* Some/all base datasets have 0 samples